### PR TITLE
Update Chinese translations

### PR DESCRIPTION
**Pull request by jinhojang6**
This Chinese translations are completed today.

### CODE REVIEW

File: themes/navy/languages/zh.yml [hunk 1]
```diff
@@ -1,648 +1,709 @@
...
-      description: Status是基于耳语协议的点对点通讯协议，使用端对端加密来保护通信不被第三方窃取，确保发送给你的消息只有你自己可见。
-    wallet:
-      subtitle: 实现社区资金
-      title: 安全加密钱包
-      description: 安全地发送、存储、接收数字资产包括ERC20以及部分ERC721数字代币，私钥掌握在你自己手中，钱包是无服务端的。直观的设计保护你免受攻击。
-    browser:
-      subtitle: 拥有你的数据
-      title: 去中心化Web3浏览器
-      description: 用兼容Web3的浏览器在日益增长的去中心化应用中冲浪，可以免受追踪，比如浏览最新的市场、交易所、游戏等。
+zh_Hans_CN:
```
Comment (jakubgs) [inline themes/navy/languages/zh.yml]: Are we sure about this root key change?

Comment (jakubgs) [inline themes/navy/languages/zh.yml]: Are we sure about this root key change?

Comment (jinhojang6) [reply themes/navy/languages/zh.yml]: @jakubgs 

My bad, I should have deleted this line.

Comment (jinhojang6) [reply themes/navy/languages/zh.yml]: Same as below

**Comment by jinhojang6** [comment]
@jakubgs Fixed it. Thank you for finding this.

**Pull request merged**